pom.xml

```diff
@@ -134,6 +134,22 @@
                 <artifactId>commons-lang3</artifactId>
                 <version>3.7</version>
             </dependency>
+            <!--
+            @todo #939:30min Upgrade to Junit5. Continue to upgrade to Junit5
+             updating qulice-maven-plugin sub-module and removing remaining
+             junit4 tags.
+            -->
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
```

Comment:
@paulodamaso you also need to upgrade surefire and failsafe to the newest versions, right now they are taken from jcabi-parent